### ISSUE TITLE
Request + Suggestion

### ISSUE THREAD

**Issue by stpnwf** [RusticBard/Neuwaita#21]
## Request:

These are some icons I would love to see in Skeuowaita

- [x] Eclipse - org.eclipse.Java / org.eclipse.Javascript
- [x] Android Studio - com.google.AndroidStudio
- [x] Pulsar - dev.pulsar_edit.Pulsar (Community maintained Atom fork)
- [ ] Warp - app.drey.Warp
- [ ] FreeTube - io.freetubeapp.FreeTube
- [x] OnlyOffice - org.onlyoffice.desktopeditors

## Suggestion:

I love the icons for Speed and Clapper. 
I find that having a lighter color in the bottom (on the gradient underneath the main circle/square) of the icons really gives them a nice 3D look and accentuates the bottom of the icon.

Feel free to totally disregard all of this advice, but I would love to see more icons like those ^.

I am unsure if I am being clear about what I mean, so here are a few examples to hopefully illustrate what I mean: 
The icons for Speed, and now most recently the icon for Clapper inspired me to do them like that. 

![jellyfin](https://user-images.githubusercontent.com/81482539/235261680-2942577a-63b6-460e-9945-d376d14212af.svg)
![matlab](https://user-images.githubusercontent.com/81482539/235261757-bfe55607-8308-48fc-87bc-9696b755b3de.svg)
![design](https://user-images.githubusercontent.com/81482539/235263095-e07bada1-f975-4d77-9062-f12e67fe9408.svg)

![timeshift](https://user-images.githubusercontent.com/81482539/235263645-8f5953c8-2481-4fae-ae46-7ff75a52926a.svg)
![syncthing](https://user-images.githubusercontent.com/81482539/235260166-7ffb8ab8-600d-4428-b77b-26d7a9282c16.svg)
![tor](https://user-images.githubusercontent.com/81482539/235260204-ce69f343-984f-4f6d-939a-81a981088567.svg)
![librewolf-0](https://user-images.githubusercontent.com/81482539/235259859-0fc63879-f85b-4654-9141-911499b99da1.svg)

**Comment by Frostbitten-jello** [comment]
The icons are designed like they are illuminated from top so why I prefer the top edge to be the lightest, but I agree that the lighter bottom edge looks really good !! Earlier I used to make the bottom edge darker than the front surface but lighter than the bottom surface but it just ended up blending it when looked at in normal usage, but I changed it and make then a little lighter or the as light as the front surface. I will update icons with time so yea keep an eye out :) and thanks for the feedback

**Comment by Frostbitten-jello** [comment]
Duplicate of #7

**Comment by stpnwf** [comment]
Yeah, no worries. Keep doing what feels right to you. I just wanted to say that.
